books/free-programming-books-langs.md

```diff
@@ -1975,6 +1979,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Code Like a Pythonista: Idiomatic Python](https://web.archive.org/web/20180411011411/http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html) - David Goodger *(:card_file_box: archived)*
 * [Composing Programs](http://composingprograms.com) (3.x)
 * [Computational and Inferential Thinking](https://inferentialthinking.com/chapters/intro.html) - Ani Adhikari, John DeNero, David Wagner (HTML)
+* [Computational Physics with Python](https://belglas.com/wp-content/uploads/2018/03/cpwp.pdf) - Eric Ayars (PDF)
```

Comment:
There's no indications that this is an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.